UIkitDZ/VC/KindPizzaVC.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  KindPizzaVC.swift
+//  UIkitDZ
+//
+//  Created by Timofey Privalov on 01.07.2021.
+//
+
+import UIKit
+
+class KindPizzaVC: UIViewController {
+
+    var recongizerDolly = 0
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.navigationBar.topItem?.title = ""
+        createPizzaImage(img: "mar.png", x: 30, y: 160)
+        createPizzaImage(img: "pep.png", x: 30, y: 360)
+        
+        createPizzaLabel(nameType: "Маргарита", x: 200, y: 210)
+        createPizzaLabel(nameType: "Пепперони", x: 200, y: 410)
+        
+        createButtonPlus(x: 330, y: 230, tag: 0)
+        createButtonPlus(x: 330, y: 430, tag: 1)
+    }
+    
+    @objc func showIngridients(sender: UIButton) {
+        let ingridientsVC = PizzaIngridients()
```

Comment:
```suggestion
        let ingridientsVC = PizzaIngridients()
```
это выше viewDidLoad и сделай приватным